src/elexmodel/models/GaussianElectionModel.py

```diff
@@ -83,6 +90,13 @@ def get_unit_prediction_intervals(self, reporting_units, nonreporting_units, alp
             lower.round(decimals=0), upper.round(decimals=0), prediction_intervals.conformalization
         )
 
+    # At the unit level, conformalization data is adjustment from estimated % change from baseline
+    def get_conformalization_data_unit(self):
+        return self.gaussian_bounds_unit, self.conformalization_data_unit
```

Comment:
I'm wondering whether it might make sense to separate out getting gaussian bounds and conformalization data to avoid confusion.

Reply:
changed function names to be 'get_all_conformalization_data_unit' [or agg] to clarify a bit. we can change later maybe